x-pack/plugins/apm/public/components/app/storage_explorer/index.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useState } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+import { IndexLifecyclePhaseSelectOption } from '../../../../common/storage_explorer_types';
+import { IndexLifecyclePhaseSelect } from './index_lifecycle_phase_select';
+import { ServicesTable } from './services_table';
+import { SearchBar } from '../../shared/search_bar';
+
+export function StorageExplorer() {
+  const [indexLifecyclePhase, setIndexLifecyclePhase] = useState(
+    IndexLifecyclePhaseSelectOption.Hot
```

Comment:
should this be a query parameter?

Reply:
It might be good to have it, thanks

Reply:
Added in https://github.com/elastic/kibana/pull/130152/commits/16efe4dab8195914759c86f2f0ab280e01e46fae